### ISSUE TITLE
[Feature Request] Add ability to change padding on langtag

### ISSUE THREAD

**Issue by samhsabin** [metonym/svelte-highlight#324]
Right now the default padding for langtag is massive, making it more of a full corner div than a "tag". Would be nice if we could adjust the padding and distance from the top right so we could make it appear like a tag.

**Comment by metonym** [comment]
What is your current workaround?

**Comment by samhsabin** [comment]
I just to not style it at all because I couldn't create something I was happy with

**Comment by metonym** [comment]
A workaround is to use global CSS to override the style. I'll look into adding a style prop to make this customization easier.

```css
<style>
  :global([data-language="css"]:after) {
    padding: 0.5rem !important;
  }
</style>
```

**Comment by samhsabin** [comment]
Thank you for looking into it 🙏

**Comment by metonym** [comment]
@samhsabin [v7.5.0](https://github.com/metonym/svelte-highlight/releases/tag/v7.5.0) now supports the `--langtag-padding` prop. You can try using that to override the default padding value.

```svelte
<HighlightAuto
  {code}
  langtag
  --langtag-padding="0.5rem"
/>
```

**Comment by samhsabin** [comment]
Thanks! You're a legend for adding that so quickly! I'll give it a test run

**Comment by samhsabin** [comment]
It works perfectly! I guess I didn’t think things through completely, but in order to make it look like a tag I would need to be able to add margin to the langtag as well, so that I can move it away from the edge, while also maintaining the background. Is there any way you could add a —langtag-margin property as well? If not, I completely understand. Either way, thanks for all your support so far.

**Comment by metonym** [comment]
@samhsabin I hear you. I exposed the `--langtag-top` and `--langtag-right` style props, which controls the positioning of the tag.

Available in [v7.6.0.](https://github.com/metonym/svelte-highlight/releases/tag/v7.6.0)

---

```svelte
<HighlightAuto
  {code}
  langtag
  --langtag-top="0.5rem"
  --langtag-right="0.5rem"
/>
```
